package.json

```diff
@@ -70,7 +70,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm run prepush"
+      "pre-push": "npm run prepush",
+      "prepare-commit-msg": "husky-add-issue-tracker"
```

Comment:
👏

Reply:
Yeah, they have this on OrgAdmin—and I like it!